### PR TITLE
Feat : Added more card components

### PR DESCRIPTION
**Pull request by kiranpratyush**
Added more card components and refactored code
Note: please ignore the inline styling for iframe it is coming by default
[Card Component](https://testing2311.netlify.app/card/index.html)

### CODE REVIEW

File: card/styles.css [hunk 1]
```diff
@@ -97,3 +130,11 @@
...
   font-size: 2rem;
 }
+
+/* This style for iframe(not working properly) */
+.iframe-style {
+  height: 500px;
+  background-color: transparent;
+  padding-left: 0px;
+  padding-right: 0px;
+}
```
Comment (BelideNikhil) [inline card/styles.css:140]: Save this as utility as you'll be using it in multiple components

**Review by BelideNikhil — APPROVED** [review]

File: card/styles.css (hunk 1 above)
Comment (kiranpratyush) [reply card/styles.css:140]: Moved to general.css Thank you for the feedback

File: card/styles.css [hunk 2]
```diff
@@ -14,17 +15,23 @@
...
   line-height: 1.25;
-  max-width: 260px;
+  max-width: 400px;
   min-width: 260px;
 }
 .card-body {
   display: flex;
   flex-direction: column;
+  align-items: space-between;
   gap: 20px;
```
Comment (sakshamAK) [inline card/styles.css:25]: Try to give the gaps and widths in rems too, to keep your values following a consistent ratio.

**Review by sakshamAK — APPROVED** [review]

File: card/styles.css (hunk 2 above)
Comment (kiranpratyush) [reply card/styles.css:25]: Thank you. 
 I will do.

**Pull request merged**